samples/CMakeLists.txt

```diff
@@ -6,6 +6,15 @@
 
 if (CONFIG_SPM)
   add_child_image(spm ${CMAKE_CURRENT_LIST_DIR}/nrf9160/spm)
+
+  if (CONFIG_ARM_FIRMWARE_USES_SECURE_ENTRY_FUNCS AND
+      NOT CONFIG_SPM_BUILD_STRATEGY_FROM_SOURCE)
+    message("NOTE: SPM is not built from source, and the firmware use secure "
+      "entry functions.
+      Ensure that the configuration 'ARM_ENTRY_VENEERS_LIB_NAME'
+      points to the .a file (default 'libspmsecureentries.a')
+      generated alongside the flashed or merged SPM hex file")
+  endif()
```

Comment:
What does the error look like when it does not point to the correct path?

Reply:
```
ninja: error: 'spm/libspmsecureentries.a', needed by 'zephyr/zephyr_prebuilt.elf', missing
 and no known rule to make it
```

So the path is grep-able, which is good.

Reply:
Let's check if there is a file at LIB_NAME and omit the warning if so. We don't want to cause noise for users that have things configured correctly.

Reply:
Added `... AND NOT EXISTS ${ARM_FILE}`